charts/portal/values.yaml

```diff
@@ -509,14 +509,16 @@ backend:
       # requests:
       #   cpu: 250m
       #   memory: 256Mi
+    seeding:
+      testDataEnvironments: ""
+      # when changing the testDataPath the processIdentity needs to be adjusted as well,
```

Comment:
Could you please use the helm-docs syntax for comments? https://github.com/eclipse-tractusx/portal-cd/blob/aca736b785342c19a58a9ae7a3d9b295463a388f/charts/portal/values.yaml#L22
Then the comment will be part of the description in the install table.

Reply:
good to know, thanks :)

- adjusted